plugins/inputs/windows_event_log/wineventlog/wineventlog.go

```diff
@@ -171,7 +159,8 @@ func (l *windowsEventLog) Open() error {
 	// Subscribe for events
 	eventHandle, err := EvtSubscribe(0, uintptr(signalEvent), channelPath, query, bookmark, 0, 0, EvtSubscribeStartAfterBookmark)
 	if err != nil {
-		fmt.Errorf("error when subscribing for events. Details: %v", err)
+		log.Printf("EvtSubscribe(), name %v, err %v", l.name, err)
```

Comment:
Do you need to prefix it with the E! in order to make it an error level log?

Reply:
Yes, good catch.